README.md

```diff
@@ -1,12 +1,296 @@
-## Build
```

Comment:
Please add some common info at the beginning like we did for [python client](https://github.com/ostis-ai/py-sc-client#readme). Specify compatibility with sc-machine versions 0.6.0 & 0.6.1